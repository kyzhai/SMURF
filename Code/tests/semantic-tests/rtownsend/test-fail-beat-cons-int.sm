/* Consing a beat onto a list of ints should fail if any of the ints are not powers of 2 */

x = 4. : [2, 4, 5, 8]
main = []
