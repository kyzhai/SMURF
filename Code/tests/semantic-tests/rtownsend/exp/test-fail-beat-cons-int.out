Fatal error: exception Sast.Type_error("The types of the lhs and rhs of a cons operator don't match")
